app/components/work_version_detail_component.rb

```diff
@@ -7,43 +7,23 @@ def initialize(work_version:)
     @work_version = work_version
   end
 
+  delegate :work, to: :work_version
+
   def render?
-    !current_draft_version? && i18n_key.present?
+    work_version.draft? || !current_representative_version?
```

Comment:
I think this needs to be

`work_version.draft? && !current_representative_version?`

As it stands, if the current version is a draft and the representative version (i.e. we only have 1 version and it's a draft), the message will show.

Reply:
But we do want that, however, it shouldn't link to the current version because it hasn't been published yet.

Reply:
![Screen Shot 2021-06-24 at 11 19 11 AM](https://user-images.githubusercontent.com/312085/123289034-17ad6580-d4de-11eb-82b2-2f9772d3f3ee.png)